content/docs/06-integrations/03-maintenance-policy/03-maintenance-policy.md

```diff
@@ -0,0 +1,174 @@
+---
+title: "Maintenance Policy"
+metaTitle: "Packs Maintenance Policy"
+metaDescription: "Learn about packs that Palette offers and choose from Pallette packs."
+icon: ""
+hideToC: true
+fullWidth: true
+hideToCSidebar: true
```

Comment:
```suggestion
hideToCSidebar: false
```
We want the right-hand table of content.

Reply:
Fixed.